apps/delphi_rest_api.cpp

```diff
@@ -87,18 +87,6 @@ class Experiment {
             result["status"] = "completed";
             vector<double> timesteps_nparr =
                 linspace(startTime, endTime, numTimesteps);
-
-            /*
-                The calculation of the 95% confidence interval about the median
-               is taken from:
-                https://www.ucl.ac.uk/child-health/short-courses-events/ \
-                about-statistical-courses/research-methods-and-statistics/chapter-8-content-8
-            */
-            int n = G.get_res();
-            int lower_rank = (int)((n - 1.96 * sqrt(n)) / 2);
-            int upper_rank = (int)((2 + n + 1.96 * sqrt(n)) / 2);
-            lower_rank = lower_rank < 0 ? 0 : lower_rank;
-            upper_rank = upper_rank >= n ? n - 1 : upper_rank;
             unordered_map<string, vector<string>> res_data;
```

Comment:
Do we need this variable and the 2 lines that follow?

I could not locate a usage of it.